lms/djangoapps/instructor_task/tasks_helper/grades.py

```diff
@@ -65,6 +65,14 @@ def _user_date_enrolled(user, course_id):
     enrollment_date = CourseEnrollment.objects.get(user_id=user, course_id=course_id).created
     return enrollment_date.strftime("%d-%m-%Y")
 
+def _unround_grade(grade):
+    """
+    Returns the grade without round
+    """
+    grade_result = str(grade)[:6]
```

Comment:
What’s a example of a value grade will contain?

Reply:
In the case of "0.16666666666666669" edX rounds to 0.17, then, we extract 6 digits, that is, 0.1666, equivalent to 16.66% in exchange for 17%.

Reply:
So you want to truncate precision to two decimals?

Reply:
Yes, because a case occurred in which one student got (5 + 5 + 5 + 4.9) / 4 = 4.97 and edX rounded it to 5 (100%). That is not appropriate in this case.

![image](https://user-images.githubusercontent.com/18407105/44067147-98fb17f0-9f39-11e8-952f-6c4d50750152.png)

Reply:
What is on line 334 makes an approximation to the number, while str (grade) [: 6] cuts it to 6 digits and on line 334, it is only to show it in format 2.45, so that the visualization is uniform.